app/pings.yaml

```diff
@@ -29,3 +29,21 @@ installation:
     - https://github.com/mozilla-mobile/fenix/pull/8074#issuecomment-586512202
   notification_emails:
     - fenix-core@mozilla.com
+
+startup-timeline:
+  description: >
+    This ping is intended to provide an understanding of startup performance. It is
+    currently not submitted to the server but is intended to be provided as a proof-
```

Comment:
How are you making sure this doesn't get submitted? Because you're calling `Pings.startupTimeline.submit()` so, if Glean is initialized and upload is enabled, it will be sent. Were you planning on making this ping only available in the "performancetest" flavour? (maybe it already is, I'm just not seeing this in this PR).

If that's the case, would you kindly be more specific in this comment?

Reply:
That was a mistake, oops! I wrote this before I added ping submission. I intend to make it available everywhere: I'll update the comment.